src/components/Input/Input.scss

```diff
@@ -1,28 +1,47 @@
 @import '~om';
 
 .Input {
-  @include text(body1);
-  line-height: 1;
-  padding: spacing(3, 4);
-  border: 1px solid color(elephant);
-  background-color: color(snow);
+  .InputInput {
```

Comment:
Why InputInput

Reply:
Basically, Input is the wrapper, InputInput is the `<input />` and `InputLabel` is the label text.

Reply:
What was wrong with `Input__input/Input__label`?

Reply:
I could also just omit `.Input` from the wrapper and make the `<input />` have the class name `.Input` instead.

Reply:
I would just observe normal BEM here. I'm not sure I'm following why you'd not want to?

Reply:
You're right, it should be Input__input and Input__label. I moved them around a couple times didn't change this back.